src/ukify/ukify.py

```diff
@@ -706,6 +706,34 @@ def make_uki(opts):
     print(f"Wrote {'signed' if sign_args_present else 'unsigned'} {opts.output}")
 
 
+UKI_DEFAULT_SECTIONS = {
+    'binary': set(['.linux', '.initrd', '.splash', '.dtb']),
+    'text' : set(['.osrel', '.uname', '.cmdline', '.pcrpkey', '.pcrsig']),
+}
+
+
+def pe_read_sections(uki: str, output: pathlib.Path):
+    pe = pefile.PE(uki, fast_load=True)
+    out = f"Sections in {uki}:\n"
```

Comment:
This is ugly and also rather ineffiecient, because the strings needs to be concatenated.
Please instead do:
```python
if output:
  cm = open(output, 'we')
else:
  cm = contextlib.nullcontext(sys.stdout)
with cm as file:
  print(…, file=file)
```
Later: support for the output file can be dropped, I think.

Reply:
As an alternative to `contextlib`, the output could also be written to an in memory file with `io.StringIO`, e.g.
```suggestion
    with io.StringIO() as outbuf:
        out.write(f"Sections in {uki}:\n")
        # all the stuff in between
        out = outbuf.getvalue()
```

Reply:
Nah, it's much nicer to use `print()` than `.write()`. That's the primary consideration I think.

Reply:
I see, I was going for efficiency here, because of the string concatenation and there `io.StringIO` is the optimum in the standard library. For using print there is also `contextlib.redirect_stdout`.